.travis.yml

```diff
@@ -1,16 +1,54 @@
 language: python
+
 python:
   - 3.6
 
+os:
+  - linux
+  - linux-ppc64le
+
+env:
+  - DOCKERFILE=Dockerfile IMAGE=max-base
+  - DOCKERFILE=Dockerfile.pai IMAGE=max-base-powerai
+
+matrix:
+  exclude:
+  - env: DOCKERFILE=Dockerfile IMAGE=max-base
+    os: linux-ppc64le
+
+dist: xenial
+
 services:
   - docker
 
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "linux-ppc64le" ]]; then
+       ARCH=ppc64le;
+    fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+       ARCH=x86_64;
+    fi
+  - mkdir -p ~/.docker
+  - echo '{"experimental":"enabled"}' | tee ~/.docker/config.json 
```

Comment:
So, the goal is to have a user do `docker pull max-base:powerai-latest` and get an x86 image on an x86 machine, and a power image on a power machine. This is only available through the use of the `manifest` command, as far as I know.

(Note that dockerhub supports this widely on official images at this point. Pretty much any OS image is multiarch, presumably created with the manifest command)

I understand the apprehension with turning on experimental functions. The manifest function was supposed to become a standard function, and that may have already happened in the newest version of docker. I'll check on this.

Reply:
As I mentioned in https://github.com/IBM/MAX-Base/pull/13#discussion_r288382167 after thinking about this, we may not have to do this for each build after all. If we manually push a manifest, I believe it will stay updated. I will verify this.

Reply:
And that's a big no! Unfortunately there's no way around using the `manifest` command if we want a multiarch image.

Reply:
As discussed in slack the manifest is necessary and this should stay as is